### ISSUE TITLE
Side panel is opened by default and system does not remember not to show it

### ISSUE THREAD

**Issue by ptoche** [texstudio-org/texstudio#3782]
The side panel is opened upon restart even after it has been de-selected via the View->Show menu. This did not happen with versions earlier than 4.8.0. A very similar issue was observed with 4.8.1.

![screenshot](https://github.com/user-attachments/assets/b34b99f2-d5bc-45f6-8127-367353bdb3ba)

### Environment
- TeXstudio: 

TeXstudio 4.8.2 (git 4.8.2)
Using Qt Version 6.7.2, compiled with Qt 6.7.2 R

- OS: 
MacBook Pro
Chip: Apple M1 Pro
Memory: 16 GB
MacOS: Sonoma 14.16.1

- TeX distribution: 
pdfTeX 3.141592653-2.6-1.40.26 (TeX Live 2024)
kpathsea version 6.4.0
Copyright 2024 Han The Thanh (pdfTeX) et al.

### Expected behavior

The side bar is not opened.
The side bar elements such as TOC, Bookmarks, etc. should not show after they have been de-selected from the 
View -> Show menu

### Actual behavior
The side bar shows up every time upon restart

### How to reproduce
Close the side bark. Restart Texstudio.

### Related issues:
These were closed, but unfortunately the problem reported there had not been resolved.

https://github.com/texstudio-org/texstudio/issues/3696
https://github.com/texstudio-org/texstudio/issues/3667

**Comment by octaeder** [comment]
can't reproduce with Win10 and latest own build (TeXstudio 4.8.3 (git 4.8.2-44-gad95c07be)).

**Comment by muzimuzhi** [comment]
Does the menu view/show/reset sidepanel helps?

**Comment by sunderme** [comment]
not reproducible in VM with OSX

**Comment by ptoche** [comment]
I see... Do you know if there is a setting in the "texstudio.ini" file that would control this behavior? I could try to fix it there. I'm seeing this:

    Structure\MarkStructureElementsBeyondEnd=true
    Structure\MarkStructureElementsInAppendix=true
    Structure\ShowElementsInComments=false
    StructureView\BackgroundColorInGlobalTOC=1
    StructureView\IndentIncludes=false
    StructureView\Indentation=-1
    StructureView\ReferenceCommandsInContextMenu=\\ref
    StructureView\ScrollToCurrentPosition=true
    StructureView\ShowLinenumbers=false
    StructureView\SingleDocMode=false

**Comment by sunderme** [comment]
Not sure if style "Fusion" is active, the icon bar on the left is too large and and does not show a scroll-indicator.
Maybe deactivate some panels to make it smaller ? ([wiki - Known Issues - osx](https://github.com/texstudio-org/texstudio/wiki/Known-Issues#osx))

**Comment by ptoche** [comment]
I went the painful way of removing the texstudio app, deleting the `.config` directory, installing again, selecting 

- Restore previous session at startup
- removing the sidebar and all its elements via the `View -> show` menu.

and it came back all the same. So it appears my `.ini` file doesn't get updated at all anymore. I'm on the same laptop. Could permissions have changed or something?

If I copy-paste a modified `.ini` file, texstudio reads it.

If I had a setting line to control the appearance of the sidebar, I could add that to my `.ini` file.

I'm looking for a line like maybe:
``
Editor\SidebarVisible=false
``

![texstudio-screenshot-1](https://github.com/user-attachments/assets/d39d971b-0b1f-4524-b46f-848499677f76)

**Comment by muzimuzhi** [comment]
What's the ownership of your `~/.config` directory? Run `ls -lah ~ | grep '\.config'` and check whether the third column shows your username.

See also https://apple.stackexchange.com/questions/320682/ownership-of-the-config-folder-in-home-directory.

**Comment by ptoche** [comment]
Thanks @muzimuzhi for getting back to me! 

I don't use any cloud storage services. I don't use OneDrive.

The ownership shows me:

  `drwxr-xr-x   15 ptoche  staff   480B Sep 14 09:38 .config`

**Comment by octaeder** [comment]
the line is `GUI\sidePanel\visible=true`

**Comment by ptoche** [comment]
Thanks @octaeder !

So I tried with both `true` and `false` and it makes no difference.

The other related settings are:

```

GUI\ConfigShorcutColumnWidth=340
GUI\Icon%20Theme=0
GUI\PDFToobarIconSize=16
GUI\SecondaryToobarIconSize=16
GUI\Style=0
GUI\SymbolSize=32
GUI\Texmaker%20Palette=false
GUI\ToobarIconSize=22
GUI\Use%20System%20Theme=true
GUI\outputView\visible=false
GUI\sidePanel\currentPage=0
GUI\sidePanel\visible=false
GUI\sidePanelSplitter\state=@ByteArray(\0\0\0\xff\0\0\0\x1\0\0\0\x2\0\0\x1\x18\0\0\x1\\\0\0\0\0\x1\x1\0\0\0\x1\0)
```

**Comment by ptoche** [comment]
With release 4.8.3 the bottom log panel remains closed (texstudio remembers the last state), while the structure side panel remains open upon a restart, whether I set `GUI\sidePanel\visible=true` or `GUI\sidePanel\visible=false` in the `texstudio.init` file.

**Comment by mgerfin** [comment]
Fixed on my Mac now, with release 4.8.3, MacOS Sequioa and the M1 version. The process of opening TexStudio for the first time is another matter with the new OS (needed to give permission in system preferences).

**Comment by ptoche** [comment]
No such luck here. 

In my `texstudio.ini' I deleted all the GUI related commands, which were:

``
GUI\ConfigShorcutColumnWidth=340
GUI\Icon%20Theme=0
GUI\PDFToobarIconSize=16
GUI\SecondaryToobarIconSize=16
GUI\Style=0
GUI\SymbolSize=32
GUI\Texmaker%20Palette=false
GUI\ToobarIconSize=22
GUI\Use%20System%20Theme=true
GUI\outputView\visible=false
GUI\sidePanel\currentPage=0
GUI\sidePanel\visible=false
GUI\sidePanelSplitter\state=@ByteArray(\0\0\0\xff\0\0\0\x1\0\0\0\x2\0\0\x1\x18\0\0\x1\\\0\0\0\0\x1\x1\0\0\0\x1\0)
``

I then restarted and proceeded to remove all the sidebar items via the menu `View -> Show`, closed the app and checked the `texstudio.ini` file. It had repopulated some of the GUI settings as follows:
``
GUI\ConfigShorcutColumnWidth=200
GUI\Icon%20Theme=0
GUI\PDFToobarIconSize=16
GUI\SecondaryToobarIconSize=16
GUI\Style=0
GUI\SymbolSize=32
GUI\Texmaker%20Palette=false
GUI\ToobarIconSize=24
GUI\Use%20System%20Theme=true
``

but upon re-opening the sidebar was back to the open state with all its elements selected. Even the log file was open. 

I pasted the single line `GUI\sidePanel\visible=false`. Nothing doing. 

Bottom line: the `GUI\sidePanel\visible=false` setting is ineffective.

**Comment by ptoche** [comment]
Nothing doing. On TeXstudio 4.8.4 (git 4.8.4)
Using Qt Version 6.7.2, compiled with Qt 6.7.2 R.

The `GUI\sidePanel\visible=false`  setting is ineffective. Toggling it true or false has no effect. Manually deselecting from `View->Show`, closing and reopening: the items are selected again:

![screenshot](https://github.com/user-attachments/assets/b7982dd1-7670-4a33-ab97-4efc51bc4766)

**Comment by ptoche** [comment]
I've been on `TeXStudio 4.8.4` for about 3 weeks (judging from the message above). I use TeXStudio every day for hours, restarting a dozen times at least every day. And tonight, all of a sudden, the system remembers to keep the side panel closed. I haven't done anything: no update, no fiddling with the `texstudio.ini file`. I've tested it a dozen times. I've compared the ini file with the one from three weeks ago and there identical word for word. Who knows what tomorrow will bring? Particularly as I haven't updated to Sequoia yet...

**Comment by ptoche** [comment]
I just like to close issues.
